tests/test_admin.py

```diff
@@ -82,7 +82,7 @@ def test_delete_selected_action_visibility(self):
         mock_request.user = self.user
         mock_request._collection = self.collection
         actions = self.mr_tree_admin.get_actions(request=mock_request)
-        self.assertIn("delete_selected", actions)
+        self.assertIn("remove_selected", actions)
```

Comment:
@jonathan-s  Going off your comment regarding the conflict with the delete actions we may now need to assert that delete_selected is not also in the actions as that is the default action.

Reply:
Fair enough. Though if it were the tests wouldn't run in the first place.

Reply:
Sorry mate, I'm not following. The delete_selected method was overriding the delete action before, this change now adds a new action called remove selected. I'm guessing now the list will contain two actions named the same but fire different actions, one fires the remove_selected action which adds a permissions check, the existing is the standard delete action which we shouldn't be able to access.

Reply:
Either way, this has been amended.